core/sql/optimizer/NAClusterInfo.cpp

```diff
@@ -350,14 +244,8 @@ NAClusterInfo::NAClusterInfo(CollHeap * heap)
         if ((nodeInfo[i].type & MS_Mon_ZoneType_Aggregation) != 0 ||
```

Comment:
I would think Edge and Aggregation should be considered as the valid nodes. Please check with @zcorrea too